.extlib/simplesamlphp/templates/post.php

```diff
@@ -4,8 +4,8 @@
 <head>
     <meta http-equiv="content-type" content="text/html; charset=utf-8" />
     <title>POST data</title>
-    <script src="/<?php echo $this->data['baseurlpath']; ?>../.extlib/simplesamlphp/www/resources/post.js"></script>
-    <link type="text/css" rel="stylesheet" href="/<?php echo $this->data['baseurlpath']; ?>../.extlib/simplesamlphp/www/resources/post.css" />
+    <script src="/<?php echo $this->data['baseurlpath']; ?>../post.js"></script>
+    <link type="text/css" rel="stylesheet" href="/<?php echo $this->data['baseurlpath']; ?>../post.css" />
```

Comment:
The original code is:
```
    <script src="/<?php echo $this->data['baseurlpath']; ?>resources/post.js"></script>
    <link type="text/css" rel="stylesheet" href="/<?php echo $this->data['baseurlpath']; ?>resources/post.css" />
```

so can we just revert back to that and put the files into here and then it should all be zero touch:

```
/auth/saml2/sp/resources/post.css
/auth/saml2/sp/resources/post.js
```

Reply:
and this should still work when we upgrade to the next version which dumps the php templates and moves to twig instead:

https://github.com/simplesamlphp/simplesamlphp/blob/master/templates/post.twig

Reply:
ah snap, didn't see the other comments, we're on the same page :)